src/index.js

```
@@ -1,183 +1,999 @@
+  bun: {
+    init: ["bun", ["init", "-y"]],
+    installDev: (dependencies) => ["bun", ["add", "--dev", ...dependencies]],
+    run: (script) => `bun run ${script}`,
+  },
+};
+
+const supportedPackageManagers = Object.keys(packageManagerCommands);
+const args = process.argv.slice(2);
+
+function exitUnsupportedPackageManager(packageManager) {
+  console.error(
```

ESLint

[error] Unexpected console statement. Only these console methods are allowed: clear, info.

(no-console)